src/bindings/js/node/src/helper.cpp

```diff
@@ -591,3 +591,19 @@ std::string buffer_to_string(const Napi::Value& value) {
 
     return std::string(reinterpret_cast<char*>(model_data.Data()), model_data.Length());
 }
+
+uint32_t get_optimal_number_of_requests(const ov::CompiledModel& actual) {
+    try {
+        auto supported_properties = actual.get_property(ov::supported_properties);
+        OPENVINO_ASSERT(
+            std::find(supported_properties.begin(), supported_properties.end(), ov::optimal_number_of_infer_requests) !=
+                supported_properties.end(),
```

Comment:
```suggestion
        const auto supported_properties = actual.get_property(ov::supported_properties);
        const auto has_optimal_num_of_requests = std::find(supported_properties.begin(), supported_properties.end(), ov::optimal_number_of_infer_requests) !=
                supported_properties.end();
        OPENVINO_ASSERT(
            has_optimal_num_of_requests,
```
Optional:
The `has_optimal_num_of_requests` will be put in error message instead the `std::find(....`